composer.json

```diff
@@ -35,7 +36,10 @@
   "autoload": {
     "files": [
       "vendor/yiisoft/yii2/Yii.php"
-    ]
+    ],
+    "psr-4": {
+      "Sil\\SspMfa\\Behat\\": "features/"
```

Comment:
Is this namespace still correct, given that those files are now here in ssp-base? Looks like now it might need to be `"SimpleSAML\\Module\\mfa\\Behat\\"`, but I'm not positive.

Reply:
Good question. Actually, I think it can be anything as long as the usage matches. I'll change it to better match where it now lives.